cmd/sops/main.go

```diff
@@ -295,7 +297,11 @@ func runEditor(path string) error {
 		}
 		cmd = exec.Command(strings.Split(string(out), "\n")[0], path)
 	} else {
-		cmd = exec.Command(editor, path)
+		parts, err := shlex.Split(editor)
```

Comment:
It uses bourne shell rules to split the string. We could achieve something that would work in most cases just splitting on spaces, but that would break if, for example, someone had something like:

```
$EDITOR=editor "foo bar"
```

I expect there are many more edge cases where simply splitting on spaces wouldn't work.